packages/connect-disputable-voting/src/thegraph/connector.ts

```diff
@@ -285,6 +287,26 @@ export default class DisputableVotingConnectorTheGraph
     )
   }
 
+  async arbitratorFee(arbitratorFeeId: string): Promise<ArbitratorFee | null> {
+    return this.#gql.performQueryWithParser<ArbitratorFee | null>(
+      queries.GET_ARBITRATOR_FEE('query'),
+      { arbitratorFeeId },
+      (result: QueryResult) => parseArbitratorFee(result, this)
+    )
+  }
+
+  onArbitratorFee(
+    arbitratorFeeId: string,
+    callback: SubscriptionCallback<ArbitratorFee | null>
+  ): SubscriptionHandler {
+    return this.#gql.subscribeToQueryWithParser<ArbitratorFee | null>(
```

Comment:
Same than above, we could use `subscription()` / `SubscriptionResult` here.

Reply:
Compared to other apps it seems like we are returning `SubscriptionHandler`s from the connectors always, is that correct?

Reply:
Yeah we might need to update other parts: `SubscriptionHandler` is now more of an internal type used by `SubscriptionResult`, which is the one we should use from subscription methods :-)